<launch>
        <arg name="leaf_size" default="1.0" />
        <arg name="radius_search" default="0.5" />
        <arg name="cluster_min_size" default="5.0" />
        <arg name="cluster_max_size" default="30.0" />

        <node pkg="ydlidar" type="ydlidar" name="ydlidar" args="input:=/scan" output="screen">
                <param name="leaf_size" value="$(arg leaf_size)"/>
                <param name="radius_search" value="$(arg radius_search)"/>
                <param name="cluster_min_size" value="$(arg cluster_min_size)"/>
                <param name="cluster_max_size" value="$(arg cluster_max_size)"/>
        </node>
</launch>
